packages/closer/locales/tdf/en.json

```
@@ -67,480 +67,181 @@
+
+  "governance_title": "Governance",
+  "governance_create_proposal": "Create Proposal",
+  "governance_proposals": "Proposals",
+  "governance_vote": "Vote",
+  "governance_draft": "Draft",
+  "governance_active": "Active",
+  "governance_closed": "Closed",
+  "governance_ready": "Ready",
+  "governance_yes": "Yes",
+  "governance_no": "No",
+  "governance_abstain": "Abstain",
```

Biome (2.1.2)

[error] The key governance_yes was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)